helpers/controllers/submissionController.ts

```diff
@@ -41,15 +42,39 @@ export const createSubmission = async (
         }
       })
     }
-    return prisma.submission.create({
+    const submission = await prisma.submission.create({
       data: {
         challengeId,
         lessonId,
         userId: id,
         diff,
         status: SubmissionStatus.Open
+      },
+      include: {
+        challenge: true,
+        user: true,
+        lesson: true
       }
     })
+
+    const { challenge, user, lesson } = submission
+
+    // TODO: Add support for discord ids when oauth implementation is complete
+
+    // Get next lesson
+    const nextLesson = await prisma.lesson.findFirst({
+      where: { order: lesson.order + 1 }
+    })
```

Comment:
Seems like we could just have used lesson.order in the `sendSubmissionNotification(...)` and then moved this id lookup to the c0d3r side.

Micro optimization but it might be a little easier to use api

Reply:
`sendLessonChannelMessage` is used here. I feel id is the better property to refer to channel's by, it is more self-explanatory in code and docs to see lessonId than lessonOrder. Are you suggesting to make a different endpoint in c0d3r api for this specific case?
This codepath is only executed when someone completes a lesson so this isn't executing too often, I think for now it is probably better to have this here than abstracting it away inside the bot.